docs/generate_extension_db.py

```diff
@@ -33,6 +33,24 @@ def IsMissing(value):
   return value == '(missing)'
 
 
+def NumReadFiltersFuzzed():
+  f = open('test/extensions/filters/network/common/fuzz/uber_per_readfilter.cc', 'r')
+  # Hack-ish! We only search the first 50 lines to capture the filters in filterNames().
+  data = ""
+  for i in range(50):
+    data += f.readline()
+  return data.count('NetworkFilterNames::get()')
+
+
+def NumRobustToDownstreamNetworkFilters(db):
+  count = 0
+  for extension, metadata in db.items():
+    # Count number of network filters robust to untrusted downstreams.
+    if 'network' in extension and metadata['security_posture'] == 'robust_to_untrusted_downstream':
```

Comment:
Suggest making this whole thing something like `return len(blah in db.items() if ...)`